src/main/views/interstitial.njk

```diff
@@ -1,45 +1,52 @@
-{% extends "template.njk" %}
-{% from "macros/common-components.njk" import goBack %}
-{% from "govuk/components/button/macro.njk" import govukButton %}
-{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
-
-{% block pageTitle %}
-    {{ title }}
-{% endblock %}
-
-{% block beforeContent %}
-    {{ goBack() }}
-{% endblock %}
-{% block content %}
-    <h1 class="govuk-heading-l">{{ title }}</h1>
-    <p class="govuk-body">{{ message }} </p>
-    <ul class="govuk-body govuk-!-padding-left-4 govuk-!-width-two-thirds">
-        {% for bullet in bullets %}
-            <li >{{ bullet }}</li>
-        {% endfor %}
-    </ul>
-    <p class="govuk-body">{{ moreMessage }}</p>
-    <p class="govuk-body">{{ legalMessage }} <a href="sign-in">{{ signInLink }}</a>.</p>
-    {% if currentLanguage === 'en' %}
-        {% set inset = '<p class="govuk-body">'+ welshMessage + '<a href="?lng=cy"> '+ languageLink + '</a>.</p>' %}
-    {% else %}
-        {% set inset = '<p class="govuk-body">'+ welshMessage + '<a href="?lng=en"> ' + languageLink + '</a>.</p>' %}
-    {% endif %}
-
-    {{ govukInsetText({
-        html: inset
-    }) }}
-    {{ govukButton({
-        text: continue,
-        href: '/view-option',
-        isStartButton: true
-    }) }}
-    <h2 class="govuk-heading-m">{{ beforeMessage }}</h2>
-    <p class="govuk-body govuk-!-font-weight-bold">{{ scoAndNI }}</p>
-    <p class="govuk-body">{{ contactMessage }}</p>
-    <ul class="govuk-body govuk-!-padding-left-4 govuk-!-width-two-thirds">
-        {% for option in bulletOptions %}
-            <li><a class="govuk-link" href="{{ option.link }}">{{ option.linkMessage }}</a> {{ option.message }}</li>
-        {% endfor %}
-    </ul>
-{% endblock %}
+{% extends "template.njk" %}
+{% from "macros/common-components.njk" import goBack %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+
+{% block pageTitle %}
+    {{ title }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukPhaseBanner({
+    tag: {
+      text: "beta"
+    },
+    html: betaHeading
+  }) }}
```

Comment:
if you have this in the template, why include it here also? this page extends the template